### ISSUE TITLE
Couldnt load cities for any country

### ISSUE THREAD

**Issue by Joly0** [ph202107/nordlist#5]
![grafik](https://github.com/ph202107/nordlist/assets/13993216/55d976ac-364b-4289-906a-8c6da830cd3c)

**Comment by Joly0** [comment]
Same for germany btw, i guess also for some other countries, wont test all of them

**Comment by ph202107** [comment]
Hi Joly0
It seems that you are getting an error message when trying to list the cities.  I tried to duplicate the problem here but everything seems to be working OK.   Could you paste the output from:

```
nordvpn version
nordvpn cities germany
```

Can you explain the steps you took to duplicate the problem?  Are you choosing the country through the "Countries" menu, and have you modified "function country_menu" or "function city_menu"?  

I noticed in your screenshot that there is a comma after "Switzerland".  If you've created a custom entry to list the cities with "function city_menu", check that you haven't accidentally put a comma after the country name.  For example:

```
$ nordvpn cities Switzerland,
We couldn’t load the list of cities. Please try again later.

$ nordvpn cities Switzerland
Zurich
```

**Comment by Joly0** [comment]
> ```
> nordvpn version
> nordvpn cities germany
> ```
> 
![grafik](https://github.com/ph202107/nordlist/assets/13993216/b81aa9ff-f826-4c53-86bc-6114a5fda800)

> Can you explain the steps you took to duplicate the problem? Are you choosing the country through the "Countries" menu, and have you modified "function country_menu" or "function city_menu"?
> 
I have not modified the functions. I ran these commands:
apt-get update && apt-get -y install --no-install-recommends git nano figlet lolcat curl jq wireguard wireguard-tools speedtest-cli iperf3 highlight
sh <(curl -sSf https://downloads.nordcdn.com/apps/linux/install.sh) -n
git clone https://github.com/ph202107/nordlist.git && cd /nordlist && chmod +x ./nordlist.sh on a fresh debian install

Afterwards i just run ./nordlist.sh and navigate to "14 - Countries" and select any Country.

> I noticed in your screenshot that there is a comma after "Switzerland". If you've created a custom entry to list the cities with "function city_menu", check that you haven't accidentally put a comma after the country name. For example:

Not sure where the comma is coming from, i just use any of these numbers
![grafik](https://github.com/ph202107/nordlist/assets/13993216/b9fe7777-aad3-4f0c-9557-4074755a1a65)

**Comment by ph202107** [comment]
Hi Joly0 thanks for the information.

It looks like for whatever reason the output of the "nordvpn countries" and "nordvpn cities <country>" commands are being listed with commas.  On Linux Mint and Ubuntu there are no commas, for example:

```
$ nordvpn cities Australia
Adelaide	Brisbane	Melbourne	Perth		Sydney
```

Also figlet and lolcat are not being recognized, since you should be seeing the colored ASCII text for "Countries" as in this screenshot:  https://github.com/ph202107/nordlist/blob/main/screenshots/3-country.list.png

Unfortunately I don't have a Debian install to test with at this time.  I will update the readme to exclude any reference to Debian.

I'm not sure what other problems there could be.  Both Mint and Ubuntu use gnome-terminal with Bash, so if that's related(?) you could try it as a test. Otherwise I don't think this script is going to work without modification to be more compliant.  Sorry about that!

**Comment by Joly0** [comment]
Hm, i'll try again with ubuntu.

I have already noticed, that lolcat isnt recognized by the nordlist script, though i am not sure why. As you can see by the commands i posted, it definitely should be installed

**Comment by ph202107** [comment]
Hi Joly0, I was able to do some testing.  I installed Debian 12 in a VM and chose the default desktop environment GNOME.  I'm not sure what happens without a desktop environment.  I installed nordvpn, nordlist, and related apps with the same commands you posted (as root).

What I found:
- the default "gnome-terminal" does not have the commas when listing countries/cities (nordvpn countries).  It seems the commas may be caused by the terminal-emulator being used.

- lolcat doesn't seem to work at all with root, but it does work with my Debian username. eg.  `$ echo "hello" | lolcat`

- I had to add my Debian username to the nordvpn group using this command:

`root@debian:-# usermod -aG nordvpn username`

and logging out didn't seem to take, so I rebooted the VM.

After rebooting and logging in as username, I did another git clone and ran the nordlist script.  It seems to be working OK, the ASCII shows up normally with colors, the Country list and city selection are working normally.  After logging in to the Nord Account (nordvpn login) I used the script to connect to VPN.  

I hope that helps!
